app/views/docs/databases.phtml

```diff
@@ -398,4 +398,501 @@ func main() async throws{
 }</code></pre>
         </div>
     </li>
-</ul>
\ No newline at end of file
+</ul>
+
+<p>The following query methods are currently supported:</p>
```

Comment:
Ah why is this change here? I think its not synced with master yet

Reply:
This is still not resolved ?

Reply:
It has been moved to https://github.com/appwrite/docs/blob/main/app/views/docs/queries.phtml

Reply:
Merge conflit probably :P This branch is ancient.